iron-collapse.html

```diff
@@ -124,6 +124,21 @@
       this._enableTransition = true;
     },
 
+    attached: function() {
+      this._contentObserver = Polymer.dom(this.$.content).observeNodes(
+        function onContentChanged(){
```

Comment:
this can be an anonymous function, but i think it takes an (unused in your case) `mutations` parameter